package.json

```diff
@@ -6,7 +6,8 @@
         "build:extensions": "yarn --cwd vscode && yarn compile:extensions",
         "compile:extensions": "NODE_OPTIONS=--max-old-space-size=8192 node ./src/compile.js",
         "bundle:extensions": "NODE_OPTIONS=--max-old-space-size=8192 node ./src/bundle.js",
-        "publish:latest": "node ./src/publish.js --tag=latest",
+        "publish": "yarn && yarn publish:latest",
+        "publish:latest": "node ./src/publish-vsix.js --tag=latest",
         "publish:next": "node ./src/publish.js --tag=next",
```

Comment:
"next" left untouched for now - we have no way yet to publish a next on open-vsx AFAIK

Reply:
Correction as per discussion with Miro below: it is actually possible to publish next versions on open-vsx, it's just not yet possible to distinguish between "next" and "latest" - they are done the same way and "publish:latest" above can be used to publish either a "latest" or "next" to open-vsx. So conceptually this is closer to "publish:openvsx" and "publish:npm". 

In any case we can keep "publish:next" above until we are sure we no longer need to publish to npm.